ui/package.json

```diff
@@ -132,23 +136,19 @@
     "ember-cli-flash": "4.0.0",
     "ember-cli-htmlbars": "^6.2.0",
     "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-mirage": "2.4.0",
-    "ember-cli-page-object": "1.17.10",
+    "ember-cli-mirage": "^3.0.2",
+    "ember-cli-page-object": "^2.1.3",
```

Comment:
These two bumps both enable embroider usage - the page object one also made the addon a v2 addon.